internal/server/statvar/golden/search_statvar/count_person.json

```diff
@@ -3845,160 +3989,16 @@
       "dcid": "Count_Person_100OrMoreYears_WhiteAloneNotHispanicOrLatino"
     },
     {
-      "name": "Population: 10th Grade, Female",
```

Comment:
I assume these deletions are due to sampling as we have a lot more Count_Person_* statvars now?